src/server/server.cc

```diff
@@ -1163,7 +1163,11 @@ void Server::GetInfo(const std::string &ns, const std::string &section, std::str
 
     if (section_cnt++) string_stream << "\r\n";
     string_stream << "# Keyspace\r\n";
-    string_stream << "# Last scan db time: " << std::put_time(&last_scan_tm, "%a %b %e %H:%M:%S %Y") << "\r\n";
+    if (last_scan_time == 0) {
+      string_stream << "# WARN: DB not scan, run `dbsize scan` command first.\r\n";
+    } else {
+      string_stream << "# Last scan DB time: " << std::put_time(&last_scan_tm, "%a %b %e %H:%M:%S %Y") << "\r\n";
+    }
```

Comment:
I wonder why it is in comment rather than an normal key-value line like `last_scan_time:0`? cc @git-hulk

Reply:
> I wonder why it is in comment rather than an normal key-value line like last_scan_time:0? cc @git-hulk

We suppose this line should be obvious enough for users since it's an annotation for the next field but seems most of them ignored this comment line.

Reply:
Ahh I see.